ci/scripts/pr.env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -euo pipefail
+
+# Set features, depending on our workflow
+# If sqlsmith files are modified, we run unit tests with sqlsmith enabled
+# by overriding RUSTFLAGS to enable sqlsmith feature.
+CHANGED=$(git diff --name-only origin/main | grep "ci/scripts/pr.env.sh\|src/tests/sqlsmith")
+if [[ -n "$CHANGED" ]]; then
+    echo "Configuring RUSTFLAGS for sqlsmith"
```

Comment:
It would be better to make it a feature instead of `cfg`? RisingWave will have to re-compile if cfg is changed, but it will reuse artifacts when configured as feature.

Reply:
For feature, we can test it by

```
cargo test --features enable_sqlsmith_unit_test
```